canvas_admin_tools/settings/base.py

```diff
@@ -136,6 +139,22 @@
     }
 }
 
+# RQ
+# http://python-rq.org/docs/
+
+RQ_QUEUES = {
+    'default': {
+        'HOST': REDIS_HOST,
+        'PORT': REDIS_PORT,
+        'DB': 0,
```

Comment:
Looks like it's the redis database number, ie. the N in `redis-cli -n N`.

https://github.com/ui/django-rq/blob/8350300ebbd2662528c985472ab78c6d9fd2280b/django_rq/queues.py#L54